# Testfunktion
def func():
    try: 
        print(x)
    except:
        print("Fehler")

# Hauptprogram
func()
x = 42
func()